rule-engine/rule-engine-demos-test/pom.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>ch.ivyteam.demo</groupId>
+  <artifactId>rule-engine-demos-test</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
+  <packaging>iar-integration-test</packaging>
+  <dependencies>
+    <dependency>
+      <groupId>ch.ivyteam.demo</groupId>
+      <artifactId>rule-engine-demos</artifactId>
+      <version>[9.1.0-SNAPSHOT,)</version>
+      <type>iar</type>
+    </dependency>
+    <dependency>
+      <groupId>com.axonivy.ivy.webtest</groupId>
+      <artifactId>web-tester</artifactId>
+      <version>9.1.0</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-params</artifactId>
+      <version>5.6.0</version>
+      <scope>test</scope>
+    </dependency>
+  </dependencies>
+  <build>
```

Comment:
do we deploy all test projects to maven artifactory? this is useless?

Reply:
Great catch, totally forgot to update this auto generated pom :)